omigami/utilities/spectrum_df_helper.py

```diff
@@ -0,0 +1,67 @@
+from typing import List, Optional, Tuple
+
+import pandas as pd
+from matchms import Spectrum
+from pandas.tests.groupby.test_value_counts import df
+
+
+class SpectrumDataFrameHelper:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def scale(spectra_df: pd.DataFrame, min=0, max=1, num_peaks=5000) -> pd.DataFrame:
+        """
+        Scale the Intensity column according to min and max values. By default, normalizes to 0 and 1.
+        """
+        df = spectra_df.copy()
+
+        min_intensity, max_intensity = df.Intensity.min(), df.Intensity.max()
+        df["Intensity"] = (df.Intensity - min_intensity) / (
+            max_intensity - min_intensity
+        ) * (max - min) + min
+
+        return df
+
+    @staticmethod
+    def filter(spectra_df: pd.DataFrame, num_peaks=5000) -> pd.DataFrame:
+        """
+        Returns a dataframe containing the largest n peaks according to num_peaks parameter
+        """
+        return df.nlargest(num_peaks, "Intensity").sort_values("m/z")
```

Comment:
this looks like it could just be a 1liner where this method is actually used

Reply:
Hm, I think this is/will be used in plotting. But the helper is mostly useful to users using a notebook, thats what I believe at least.

Reply:
This is used so you can show just peaks from a certain m/z range